Hackney.Shared.HousingSearch.Tests/Hackney.Shared.HousingSearch.Tests.csproj

```diff
@@ -11,6 +11,11 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="coverlet.collector" Version="1.2.0" />
```

Comment:
This is also quite old so it might be worth upgrading, from the looks of it its a code coverage package so a later version might be more accurate.

Reply:
Done in [59b2269](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/59b2269880f703294f5ae70c41430791b005b513).